app/league/[leagueId]/entry/all/page.tsx

```diff
@@ -87,13 +86,17 @@ const Entry = ({
     <>
       {entries.map((entry) => {
         const linkUrl = `/${LEAGUE_URL}/${leagueId}/${ENTRY_URL}/${entry.$id}/${WEEK_URL}/${currentWeek}`;
+        const isPickSet = entry.selectedTeams.length > 0;
+        const teamLogo = isPickSet ? entry.selectedTeams[0].teamLogo : '';
 
         return (
           <section key={entry.$id}>
             <LeagueEntries
               key={entry.$id}
               entryName={entry.name}
+              isPickSet={isPickSet}
```

Comment:
For learning purposes only, if the property and the value are the same, you can just use the variable name for shorthand. `isPickSet`

Reply:
> For learning purposes only, if the property and the value are the same, you can just use the variable name for shorthand. `isPickSet`

This is good to know!

Reply:
Good to know. I modified it to the variable name for shorthand.